exporter/exporterhelper/common.go

```diff
@@ -232,3 +234,8 @@ func (ts *timeoutSender) send(req request) error {
 	}
 	return req.export(ctx)
 }
+
+// DefaultUserAgent User-Agent value used by exporters
+func DefaultUserAgent(info component.BuildInfo) string {
```

Comment:
let's move this to otlpexporter for the moment or an internal package. Prefer to limit the public API.

Reply:
✅ Moved to each exporter https://github.com/open-telemetry/opentelemetry-collector/pull/3970/commits/f5bddaf5d217f650fa1bbd13990e43318c15a3d2